tests/test_migrators.py

```diff
@@ -166,3 +368,29 @@ def test_js_migration2(tmpdir):
     js.migrate(tmpdir, {})
     with open(os.path.join(tmpdir, 'meta.yaml'), 'r') as f:
         assert f.read() == correct_js
+
+
+def test_version_migration(tmpdir):
+    # Test meta.yaml with one url
+    with open(os.path.join(tmpdir, 'meta.yaml'), 'w') as f:
+        f.write(one_source)
+    v = Version()
+    v.migrate(tmpdir, {'new_version': '2.4.1'})
+    with open(os.path.join(tmpdir, 'meta.yaml'), 'r') as f:
+        assert f.read() == updated_one_source
+
+    # Test meta.yaml with jinja variable for sha
+    with open(os.path.join(tmpdir, 'meta.yaml'), 'w') as f:
+        f.write(jinja_sha)
+    v = Version()
```

Comment:
Can these all use the same `Version` instance? I think the migrations all use the same object.